jobs/build/operator-sdk_sync/Jenkinsfile

```diff
@@ -126,7 +128,7 @@ pipeline {
                         dir("./${arch}") {
                             sshagent(['aos-cd-test']) {
                                 sh "ssh use-mirror-upload.ops.rhcloud.com -- mkdir -p /srv/pub/openshift-v4/${arch}/clients/operator-sdk/${params.OCP_VERSION}"
-                                sh "scp -- *.tar.gz use-mirror-upload.ops.rhcloud.com:/srv/pub/openshift-v4/${arch}/clients/operator-sdk/${params.OCP_VERSION}"
+                                sh "rsync -av --no-g --progress -e \"ssh -o StrictHostKeyChecking=no\" *.tar.gz use-mirror-upload.ops.rhcloud.com:/srv/pub/openshift-v4/${arch}/clients/operator-sdk/${params.OCP_VERSION}/"
```

Comment:
Would like to remove `-e \"ssh -o StrictHostKeyChecking=no\"`.

Reply:
Sorry, was that a comment on this PR or a general wish for this repo?  It's used elsewhere so I thought it was necessary.

Reply:
@yselkowitz yeah, you are right it is used in a lot of places. I think we are good to merge this PR then I will create another PR to remove those `StrictHostKeyChecking` stuffs.

Reply:
https://github.com/openshift/aos-cd-jobs/pull/2721